scripts/bundle.js

```diff
@@ -0,0 +1,42 @@
+const {resolve} = require('path');
+const esbuild = require('esbuild')
+const {externalGlobalPlugin} = require('esbuild-plugin-external-global')
+
+const INPUT = process.argv[2]
+const OUTPUT = process.argv[3]
+
+const PACKAGE_INFO = require(resolve(INPUT, '../package.json'));
+
+/**
+ * peerDependencies are excluded using `externals`
+ * https://webpack.js.org/configuration/externals/
+ * e.g. @deck.gl/core is not bundled with @deck.gl/geo-layers
+ */
+function getExternals(packageInfo) {
+  let externals = {
+    // Hard coded externals
+    // 'h3-js': 'h3'
+  };
+  const {peerDependencies = {}} = packageInfo;
+
+  for (const depName in peerDependencies) {
```

Comment:
We've got the hubble packages in main's dependencies. Should they actually be in peerDependencies?

Reply:
So if something is in dependencies, then it's bundled with. If something is in dependencies, then it's bundled without. The difference is 

```html
<script src="https://unpkg.com/deck.gl@^8.0.0/dist.min.js"></script>
<script>
  // core, layers, and geo-layers are dependencies of main
  const {MVTLayer} = deck;
</script>
```

vs

```html
<script src="https://unpkg.com/@deck.gl/core@^8.0.0/dist.min.js"></script>
<script src="https://unpkg.com/@deck.gl/layers@^8.0.0/dist.min.js"></script>
<script src="https://unpkg.com/@deck.gl/geo-layers@^8.0.0/dist.min.js"></script>
<script>
  // core and layers are peer dependencies of geo-layers
  const {MVTLayer} = deck;
</script>
```

Reply:
I'm confused, typo? Which is peer dependencies?

Reply:
When A is in B's peer dependencies, you need to explicitly include both bundle A and bundle B for B to work.
When A is in B's dependencies, you only need to include bundle B for B to work.